CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # edge-react-gui
 
+## 2.14.0 (2022-04-26)
+
+- Add Tomb Swap
+- Add Tomb Staking v3
```

Comment:
Don't call it `Tomb Staking v3`, ask Sam/Jon which assets/staking policies they added and add all of them to the logs.